src/accelerate/state.py

```diff
@@ -903,7 +903,9 @@ def __init__(self, gradient_accumulation_plugin: Optional[GradientAccumulationPl
             self.sync_gradients = True
             self.active_dataloader = None
             self.dataloader_references = [None]
-            self.plugin_kwargs = gradient_accumulation_plugin.to_kwargs()
+            self.plugin_kwargs = (
+                gradient_accumulation_plugin.to_kwargs() if gradient_accumulation_plugin is not None else {}
```

Comment:
not strictly related but I could not test as with the added test without this change.